.github/workflows/daily.yml

```diff
@@ -0,0 +1,39 @@
+name: ci
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  daily:
+    runs-on: ubuntu-latest
+    if: contains(github.ref, 'refs/heads/master')
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
```

Comment:
1.16.x?

Reply:
`1.x` ?

Reply:
I think that's a bit overkill :) I'd rather do major versions explicitly. Bugfix versions basically never break stuff, but major releases sometimes do.

Reply:
I didn't get this, do you mean to pin it on specific version or anything in 1.16 range? If the latter `go-version: 1.16` would be `>= 1.16, < 1.17`.

Reply:
I mean literally the 1.16.x string, which will do the latest bugfix release.

Reply:
My understanding was that `1.16` is equivalent of `1.16.x`. I'll update regardless.

Reply:
Maybe, though that's a bit confusing as Go calls the first release 1.16, not 1.16.0. The ".x" helps disambiguate.